devel/asio/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                chriskohlhoff asio 1.12.0 asio-
+git.branch                  [strsed [join ${github.tag_prefix}]${github.version}[join ${github.tag_suffix}] {g/\./-/}]
+license                     Boost-1
+description                 Asio C++ Library.
+long_description            Asio is a cross-platform C++ library for network and low-level I/O programming that provides developers with a \
+                            consistent asynchronous model using a modern C++ approach.
+homepage                    http://www.think-async.com
+maintainers                 @Lord-Kamina
```

Comment:
Please add your email address here. Also can `openmaintainer` be added? Like this:

`maintainers                 {obfuscated-email-address @Lord-Kamina} openmaintainer`